src/app/app.interface.ts

```diff
@@ -55,6 +55,13 @@ export type AppDefinitionObject = {
   supportedNetworks: { [N in Network]?: ProtocolAction[] };
   primaryColor?: string;
   url: string;
+  links: {
```

Comment:
We also wanted a way to link to Learn articles, could we add a entry for it?

Reply:
Absolutely. I can add `learn` as a link.